tests/next_tests/integration_tests/multi_feature_tests/iterator_tests/test_hdiff.py

```diff
@@ -78,7 +78,12 @@ def hdiff(inp, coeff, out, x, y):
 
 def test_hdiff(hdiff_reference, program_processor_no_dace_exec, lift_mode):
     program_processor, validate = program_processor_no_dace_exec
-    if program_processor == run_gtfn or program_processor == run_gtfn_imperative:
+    if program_processor in [
+        gtfn_cpu.run_gtfn,
+        gtfn_cpu.run_gtfn_imperative,
+        gtfn_cpu.run_gtfn_with_temporaries,
+    ]:
+        # TODO(tehrengruber): check if still true
```

Comment:
This TODO should be resolved in this PR, either it's true, then add a more precise description, or it works and remove the xfail.